src/test/java/it/gov/pagopa/pu/classification/service/BalanceUnmashallerServiceTest.java

```diff
@@ -25,7 +25,7 @@ class BalanceUnmashallerServiceTest {
   private Resource resource;
   private BalanceUnmashallerService handler;
   private XMLUnmarshallerService xmlUnmarshallerService;
-  private static final String XML_STRING = "<bilancio  xmlns=\"http://www.regione.veneto.it/schemas/2012/Pagamenti/Ente/\">" +
+  private static final String XML_STRING = "<bilancio>" +
```

Comment:
we need to test 2 xml, one with the namespace, one without, not just without namespace